modules/statics/src/coins.ts

```diff
@@ -11990,6 +12054,30 @@ export const coins = CoinMap.fromCoins([
     '0x4e7e8579a9edc6283011be942537baf9284bebe1',
     UnderlyingAsset['bsc:city']
   ),
+  bscToken(
+    '5c7702c4-8eb8-423b-ab23-eff15a6a8a1c',
+    'bsc:gft',
+    'Gifto',
+    18,
+    '0x72fF5742319eF07061836F5C924aC6D72c919080',
+    UnderlyingAsset['bsc:gft']
+  ),
+  bscToken(
+    '71861b2e-03b0-4101-81ea-ea35b13d48d1',
+    'bsc:glmr',
+    'Moonbeam',
+    18,
+    '0x76F3CE6aF26de7A9854dBD153ACd8f46a2Cf5133',
+    UnderlyingAsset['bsc:glmr']
+  ),
+  bscToken(
+    'bcc8389e-a3bf-493d-89a8-3d95676e817c',
+    'bsc:gmt',
+    'Green Metaverse Token',
+    18,
```

Comment:
decimal value is 8 as per explorer: [0x3019BF2a2eF8040C242C9a4c5c4BD4C81678b2A1](https://bscscan.com/token/0x3019BF2a2eF8040C242C9a4c5c4BD4C81678b2A1)

Reply:
done